src/library/Uno.Material/Styles/Controls/Flyout.xaml

```diff
@@ -0,0 +1,528 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:winui="using:Microsoft.UI.Xaml.Controls"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:not_win="http://uno.ui/not_win"
+					mc:Ignorable="not_win">
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="../Application/Colors.xaml" />
+	</ResourceDictionary.MergedDictionaries>
+
+	<x:String x:Key="CheckGlyphPathStyle">M30.561941,0L31.997,1.393004 10.467954,23.597999 0,15.350999 1.2379759,13.780992 10.287961,20.909952z</x:String>
+	<x:String x:Key="RightArrowPathStyle">M0,0L25.194,16 0,32z</x:String>
+
+	<!-- Material FlyoutPresenter Style -->
+	<Style x:Key="MaterialFlyoutPresenterStyle"
+		   TargetType="FlyoutPresenter">
+
+		<Setter Property="Foreground"
+				Value="{StaticResource MaterialOnBackgroundBrush}" />
```

Comment:
I think flyouts are "surface", not "background. So I would use MaterialOnSurfaceBrush and MaterialSurfaceBrush